### PR TITLE
GH-688 Implemented summary card in dashboard

### PR DESCRIPTION
**Pull request by billoosijok**
Implemented summary view in the dashboard

### CODE REVIEW

File: app/src/Pages/Portal/Dashboard/cards/SummaryCard.js [hunk 1]
```diff
@@ -0,0 +1,216 @@
...
+      <div className="columns">
+        <div className="column">
+          <h5>Contributions</h5>
+          <p className="top-section">
+            <Link to="/contributions">
+              Total: {data.totalContributions.total}
+            </Link>
+            {delete data.totalContributions.total}
+            {Object.keys(data.totalContributions).map(key => (
+              <Link to={`/contributions?status=${key.toLowerCase()}`}>
```
Comment (kdv24) [inline app/src/Pages/Portal/Dashboard/cards/SummaryCard.js]: I'm getting this error `Each child in a list should have a unique "key" prop.` despite there being a key.

Comment (billoosijok) [reply app/src/Pages/Portal/Dashboard/cards/SummaryCard.js]: I pushed a fix that I think would fix this. Can you please pull and check?

Comment (kdv24) [reply app/src/Pages/Portal/Dashboard/cards/SummaryCard.js]: @billoosijok as long as it's okay that `draft` and `total` numbers are the same for now, this looks good. 👍

Comment (billoosijok) [reply app/src/Pages/Portal/Dashboard/cards/SummaryCard.js]: Yeah that's what's coming from the backend for now

**Comment by kdv24** [comment]
Looks good now, @billoosijok 👍

**Pull request merged**